src/containers/shared/components/Streams.jsx

```diff
@@ -94,6 +95,29 @@ const formatLedgers = (data) =>
     })
     .sort((a, b) => b.ledger_index - a.ledger_index)
 
+export const fetchNegativeUNL = async (rippledSocket) =>
+  getNegativeUNL(rippledSocket)
+    .then((data) => {
+      if (data === undefined) throw new Error('undefined nUNL')
```

Comment:
Why explicitly `undefined`

Reply:
I do not know.  This counts for null and undefined.